flows/flows.py

```diff
@@ -211,11 +214,15 @@ def encode_categorical_feature(self, dataframes_dict: dict, print_results: Union
 
         if not _reference:
             _reference = list(dataframes_dict.keys())[0]
+
         print(f"The reference dataframe is: {_reference}")
 
         string_columns = self.columns_set[_reference]["categorical_string"] + self.columns_set[_reference][
             "categorical_integer"]
 
+        if ignore_columns is not None:
+            string_columns = [col_i for col_i in string_columns if col_i not in ignore_columns]
```

Comment:
This code is fine, but alternatively you can express the same computation as Set Difference, which might be a bit more concise, for example:

```
>>> columns = set(['col1', 'col2', 'col3'])
>>> ignored = set(['col2', 'col4'])
>>> final = columns - ignored
>>> print(final)

{'col1', 'col3'}
```

Reply:
Another alternative, maybe, is to implement a function that accepts two Collection objects (lists, tuples ... etc) and returns the difference as a set. Could be useful, but I'll leave it for you to decide.

Reply:
As we discussed, the order of the columns should be taken into account. Therefore, "set " could be not the best solution here.

Reply:
As discussed, maybe you could use some performance improvements from turning the `ignored_columns` into a Set which is a better fit for checking membership of an item.